charts/actions-runner-controller/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.3
+version: 0.23.4
```

```suggestion
version: 0.23.3
```

It would be awesome if you could let me defer the next chart patch release for a while,
maybe until we cut a patch release for controller 0.27.x!

---

NVM. We have no pending chart fixes awaiting merging today so we can cut a release after this!
On the next occasion, I'd appreciate it if you could leave the version field as-is!

---

Awsome, thanks. I was actually trying to find if I should bump the version. I've looked at the contribution file, but couldn't find it. Today I find that you state this in https://github.com/actions/actions-runner-controller/blob/master/CONTRIBUTING.md#helm-version-changes already 🙈 😄